tests/syslog/syslog_utils.py

```diff
@@ -0,0 +1,167 @@
+"""
+    Helpful utilities for writing tests for the syslog feature.
+"""
+import logging
+import time
+import os
+
+
+DUT_PCAP_FILEPATH = "/tmp/test_syslog_tcpdump_{vrf}.pcap"
+DOCKER_TMP_PATH = "/tmp/"
+
+
+def add_syslog_server(dut, syslog_server_ip, source=None, vrf=None, port=None):
+    """
+    Add syslog server
+
+    Args:
+        dut (SonicHost): The target device
+        syslog_server_ip (str): Syslog server address
+        source (str): Source ip address
+        vrf (str): Vrf device (default,mgmt,Vrf-data)
+        port (str): Server udp port
+
+    """
+    cmd_add_syslog_server = 'sudo config syslog add {} '.format(syslog_server_ip)
+    if source:
+        cmd_add_syslog_server = "{} --source {} ".format(cmd_add_syslog_server, source)
+    if vrf:
+        cmd_add_syslog_server = "{} --vrf {} ".format(cmd_add_syslog_server, vrf)
+    if port:
+        cmd_add_syslog_server = "{} --port {} ".format(cmd_add_syslog_server, port)
+    logging.debug("add_syslog_server command is: {}".format(cmd_add_syslog_server))
+    return dut.command(cmd_add_syslog_server, module_ignore_errors=True)
+
+
+def del_syslog_server(dut, syslog_server_ip):
+    """
+    Del syslog server
+
+    Args:
+        dut (SonicHost): The target device
+        syslog_server_ip (str): Syslog server ip
+    """
+    dut.command('sudo config syslog del {} '.format(syslog_server_ip))
+
+
+def show_syslog(dut):
```

Comment:
@JibinBao please add a common `ansible/pytest` module to fetch a syslog facts

Reply:
we can move the function to common/device/sonic.py. Is it ok?

Reply:
@JibinBao you are the decision maker

Reply:
ok. I will move it to the common place.